src/PerfView/EventViewer/EventWindow.xaml

```diff
@@ -79,6 +79,7 @@
             <Menu DockPanel.Dock="Top" Background="{DynamicResource ControlDefaultBackground}">
                 <MenuItem Header="_File">
                     <MenuItem Header="_Parent Window" Click="DoOpenParent"/>
+                    <MenuItem Header="_Show Local Time" IsCheckable="True" IsChecked="False" Checked="DoUseLocalTime" Unchecked="DoUseOriginTime"/>
```

Comment:
Suggestion: Create a new "View" or "Options" menu for this. It's not a "File" thing.